app/views/home/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<!-- in app/views/home/index.html.erb -->
+<%= javascript_pack_tag 'application' %>
+<div id="root"></div>
+<script>
+  ReactDOM.render(
+    React.createElement(<%=j "#{App}" %>),
```

Comment:
Well done implementing a hello rails app with rails 👏🏾 

1. In `app/views/home/index.html.erb` file there's a syntax error with rendering your React app.

- The closing `</script>` tag is missing and the closing parenthesis `)` for the `ReactDOM.render()` is also missing.

- There's also an error with rendering `App` component for the React UI even after I fixed the syntax error. Note: sometimes Rails behaves differently on other systems, so if the render works fine on your side, you can ignore this part.

![image](https://github.com/EddyShimwa/hello-rails-react/assets/38088818/75597c46-b3cb-41e4-897e-df81bc92cd30)

[OPTIONAL] I decided to move `ReactDOM.render()` to `app/javascript/application.js` and the application worked. However, you are not obligated to do the same. What's important is that your application works with your implementation.

![image](https://github.com/EddyShimwa/hello-rails-react/assets/38088818/3679aee4-75d2-4e11-8685-40ce3b16be86)

2. After successfully rendering your application, the random message is not displayed on the `home#index` page. I checked your API endpoint  `http://localhost:3000/api/v1/greetings` which should return a JSON object with a random message and I got an error. Therefore, the issue seems to be with fetching the message from the API.

![image](https://github.com/EddyShimwa/hello-rails-react/assets/38088818/42047bc4-dad0-40cb-8cec-7fbe8247fd8e)

- Kindly look into it and make sure that your API endpoint works correctly so that when data if fetched it is rendered successfully on the React Greeting component.